lasagne/tests/layers/test_conv.py

```diff
@@ -191,7 +197,7 @@ def test_defaults(self, Conv2DImpl, DummyInputLayer,
                 W=kernel,
                 **kwargs
             )
-            actual = layer.get_output_for(input).eval()
+            actual = get_output(layer, input).eval()
```

Comment:
That's a good point, I guess there is no need to test more than `get_output_for()` here.